pandas/tests/dtypes/test_concat.py

```diff
@@ -88,3 +88,12 @@ def test_concat_mismatched_categoricals_with_empty():
     result = _concat.concat_compat([ser1._values, ser2._values])
     expected = pd.concat([ser1, ser2])._values
     tm.assert_categorical_equal(result, expected)
+
+
+# This will raise a ValueError issue if it fails
```

Comment:
Can you move this comment to the body of the function?

Reply:
resolved